src/applications/disability-benefits/all-claims/pages/alternateNames.js

```diff
@@ -0,0 +1,42 @@
+import FullNameField from 'us-forms-system/lib/js/fields/FullNameField';
+import fullSchema from '../config/schema';
+
+const { alternateNames: alternateNamesSchema } = fullSchema.properties;
+
+export const uiSchema = {
+  'view:hasAlternateName': {
+    'ui:title': 'Did you serve under another name?',
+    'ui:widget': 'yesNo'
+  },
+  alternateNames: {
+    'ui:description': 'What names did you serve under?',
+    'ui:options': {
+      viewField: FullNameField,
+      reviewTitle: 'Other names',
+      expandUnder: 'view:hasAlternateName',
+      itemName: 'Name'
```

Comment:
Looks like `ArrayField` doesn't support `addAnotherLabel`, but it does support `itemName` instead, which basically amounts to the same thing:
https://github.com/usds/us-forms-system/blob/2410134ffda3767058ae7ef16e512e3fc5124632/src/js/fields/ArrayField.jsx#L294